src/tqdm_publisher/publisher.py

```diff
@@ -9,22 +9,89 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.callbacks = {}
 
+
     # Override the update method to call callbacks 
-    def update(self, n=1, always_callback=False):
-        if super().update(n) or always_callback:
+    def update(self, n=1):
+        super().update(n)
```

Comment:
```suggestion
        super().update(n)

```

Minor styling thing; adding one line of pad to make injection point more visually obvious